src/status_im/contexts/profile/logout/events.cljs

```diff
@@ -0,0 +1,49 @@
+(ns status-im.contexts.profile.logout.events
+  (:require [native-module.core :as native-module]
+            [status-im.db :as db]
+            [utils.re-frame :as rf]))
+
+(rf/reg-fx :profile.logout/native-module native-module/logout)
```

Comment:
`:profile.logout/native-module` name is strange, if thats because of the same name, we have `effects` namespace, something like `:effects.profile/logout`